texas-hearing-institute/App.tsx

```diff
@@ -1,21 +1,70 @@
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { Pressable, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { Button } from 'react-native-elements';
```

Comment:
remove unused imports Button and Icon (we'll add a linter soon that will take care of this automatically haha)